src/Image.php

```diff
@@ -385,9 +385,6 @@ public function deleteDynamicMetadata($dynamicMetadataName, $hash, $organization
         $response = $this->call('DELETE', $path);
 
         if ('204' == $response->getStatusCode()) {
-            return $hash;
-        }
-        if ('201' == $response->getStatusCode()) {
             return $this->extractHashFromLocationHeader($response->getHeader('Location'));
```

Comment:
does the hash change when we change dynamic metadata? is that the thing that is not user defined metadata, so it actually changes the hash?